src/Native/Unix/System.Security.Cryptography.Native/pal_ecc_import_export.cpp

```diff
@@ -108,8 +108,16 @@ extern "C" int32_t CryptoNative_GetECKeyParameters(
 
     if (includePrivate)
     {
-        *d = const_cast<BIGNUM*>(EC_KEY_get0_private_key(key));
-        *cbD = BN_num_bytes(*d);
+        const BIGNUM* const_bignum_privateKey = EC_KEY_get0_private_key(key);
+        if (const_bignum_privateKey)
```

Comment:
I believe our C++ style says to always compare against nullptr, and not do the implicit Boolean.

The compiler can easily optimize that down.